pyartcd/pyartcd/pipelines/ocp4.py

```diff
@@ -887,4 +889,24 @@ async def ocp4(runtime: Runtime, version: str, assembly: str, data_path: str, da
         skip_plashets=skip_plashets,
         mail_list_failure=mail_list_failure,
     )
-    await pipeline.run()
+
+    if ignore_locks:
+        await pipeline.run()
+
+    else:
+        # Create a Lock manager instance
+        lock_policy = locks.LOCK_POLICY['ocp4']
+        lock_manager = locks.new_lock_manager(
+            internal_lock_timeout=lock_policy['lock_timeout'],
+            retry_count=lock_policy['retry_count'],
+            retry_delay_min=lock_policy['retry_delay_min']
+        )
+        lock_name = f'github-activity-lock-{version}'
```

Comment:
This looks like it can be a constructor? Also keeping the lock name at one single place

Reply:
Makes sense. I plan to open a refactoring PR to make it all clearer. I prefer not to pollute this change with the next steps though